app/serializers/api/v1/order_shallow_serializer.rb

```diff
@@ -9,6 +10,12 @@ class OrderShallowSerializer < ApplicationSerializer
       :dispatch_started_by_id, :submitted_at, :submitted_by_id, :people_helped, :beneficiary_id,
       :address_id, :district_id, :booking_type_id, :staff_note, :cancellation_reason_id, :shipment_date
 
+    has_many :messages, serializer: MessageSerializer
```

Comment:
this will include all messages, including ones with `is_private` set to true. Need to send messages separately to order payload... in admin and donor app we use the `api/v1/messages` endpoint. Same here perhaps?

Reply:
Confirmed with swati that we have a default scope to set `is_private: false`